incubator/anchore-engine/templates/NOTES.txt

```diff
@@ -0,0 +1,20 @@
+To use Anchore Engine you need the URL, username, and password to access the API.
+
+Here are the steps to configure the anchore-cli (`pip install anchorecli`). Use these same values for direct API access as well.
+
+To configure your anchore-cli run:
+
+    ANCHORE_CLI_USER=admin
+    ANCHORE_CLI_PASS=$(kubectl get secret --namespace {{ .Release.Namespace }} {{ template "fullname" . }} -o jsonpath="{.data.adminPassword}" | base64 --decode; echo)
+
+To get the service endpoint:
+
+If using an ingress:
```

Comment:
Can you conditionally template this in/out based on whether the ingress is enabled?